cypress/e2e/pages/articles/index.cy.js

```diff
@@ -188,12 +188,6 @@ const nonSmokeCanonicalTestSuites = [
     service: 'nepali',
     tests: [...canonicalTests],
   },
-  {
-    path: '/polska',
-    runforEnv: ['test'],
-    service: 'polska',
-    tests: [...canonicalTests],
-  },
   {
     path: '/polska/articles/c639526lxlro',
     runforEnv: ['test'],
```

Comment:
Should we also run this on local to catch errors? I was also thinking that we could add LDP tags to the article on test Optimo, so that these tests would pass?

Reply:
Ah yeah good shout, i can ask in the service launches for someone to add a tag to it - i went for just removing it as i wondered if it was best to wait and test a 'real' article for completeness when we have it, but we can just replace this one with a 'real' one when we get it tbf

Reply:
tags have been added, ill add the test back!

Reply:
Nice! Thanks for following up 🐱